src/controllers/midi/midicontroller.cpp

```diff
@@ -159,42 +159,49 @@ void MidiController::destroyOutputHandlers() {
     }
 }
 
-QString formatMidiMessage(unsigned char status, unsigned char control, unsigned char value,
-                          unsigned char channel, unsigned char opCode) {
+QString formatMidiMessage(unsigned char status, unsigned char control,
+                          unsigned char value, unsigned char channel,
+                          unsigned char opCode, mixxx::Duration timestamp) {
     switch (opCode) {
         case MIDI_PITCH_BEND:
-            return QString("MIDI status 0x%1: pitch bend ch %2, value 0x%3")
+            return QString("MIDI status 0x%1: pitch bend ch %2, value 0x%3, timestamp %4")
```

Comment:
Yea, I didn't notice your PR added timestamps too until I went in to suggest we collect some timestamp data :). The way you did it with 'ts XXX' as a prefix seems nice.